lua/neogit/lib/popup/init.lua

```diff
@@ -564,7 +564,7 @@ function M:show()
     kind = config.values.popup.kind,
     mappings = mappings,
     after = function(buf, _)
-      vim.cmd([[setlocal nocursorline]])
+      vim.cmd([[setlocal nocursorline nolist]])
```

Comment:
Why not 
```suggestion
      vim.opt_local.list = false
```
?

Reply:
Yea, I was thinking something like that would be nice. Decided to go with this since we have the window ID as the second function arg 🤷🏼 Which is.... more... specific?

```lua
      vim.api.nvim_set_option_value("cursorline", false, { win = win })
      vim.api.nvim_set_option_value("list", false, { win = win })
```

Reply:
double win/win even! :) Thanks for that change!